torch/testing/_internal/common_device_type.py

```diff
@@ -275,6 +277,14 @@ def setUpClass(cls):
 if torch.cuda.is_available():
     device_type_test_bases.append(CUDATestBase)
 
+
+_PYTORCH_TEST_PATH = os.environ.get('PYTORCH_TEST_PATH', None)
```

Comment:
We should make the environment variable more descriptive. Like... TORCH_TEST_DEVICES?